lib/src/components/NxPagination/__tests__/NxPagination.test.tsx

```diff
@@ -252,9 +252,9 @@ describe('NxPagination', function() {
   });
 
   describe('accessibility', function() {
-    it('includes the current and max page in the nav label and can be tabbed to for screen readers', function() {
+    it('includes an aria-label in the nav and can be tabbed to for screen readers', function() {
       const wrapper = getShallowComponent({ pageCount: 3, currentPage: 0 });
-      expect(wrapper).toHaveProp('aria-label', 'pagination, page 1 of 3');
+      expect(wrapper).toHaveProp('aria-label', 'pagination');
       expect(wrapper).toHaveProp('tabIndex', 0);
```

Comment:
I thought we were going to get rid of the tabindex as well

Reply:
> I thought we were going to get rid of the tabindex as well

Removed here: 5ef8c099f8735e252e3eb9c4d6b772f16b105f94